docs/configuration.md

```diff
@@ -610,8 +610,8 @@ Options:
           Enable preview mode; checks will include unstable rules and fixes.
           Use `--no-preview` to disable
       --extension <EXTENSION>
-          List of mappings from file extension to language (one of ["python",
-          "ipynb", "pyi"]). For example, to treat `.ipy` files as IPython
+          List of mappings from file extension to language (one of `"python"`,
+          `"ipynb"`, `"pyi"`). For example, to treat `.ipy` files as IPython
```

Comment:
nit: same as above, do we need both quote and the back tick?

Reply:
I think the quotes are intentional to make it clear that these are string values. The backticks ensure that they're rendered as code.